workshops/views.py

```diff
@@ -1986,6 +1986,8 @@ def all_activity_over_time(request):
     context = {
         'api_endpoint': reverse('api:reports-all-activity-over-time'),
         'title': 'All activity over time',
+        'start': datetime.date.today() - datetime.timedelta(days=365),
```

Comment:
why not datetime.timedelta(years=1) ? (quick check) ah - because the class doesn't support that. OK, looks good to me - @pbanaszkiewicz ?